hugo/layouts/partials/site/head.html

```diff
@@ -5,7 +5,7 @@
 {{ else }}{{ with .Site.Params.description -}}{{- . | $.RenderString | plainify -}}
 {{ end }}{{ end }}{{ end -}}">
 {{ hugo.Generator }}
-{{ if and (eq (getenv "HUGO_ENV") "production") (ne .Params.noindex true) (ne .Type "examples") }}
+{{ if and (eq (getenv "HUGO_ENV") "production") (ne .Params.no_index true) (ne .Type "examples") }}
```

Comment:
In the current tip of alpha:

```
$ git grep no_index | wc -l
3
$ git grep noindex | wc -l
16
```

I suggest either splitting this PR into 2 (moving the noindex fix into its own PR, with fixups for all the current in-repo uses of `noindex`), or including all those fixups in this PR.

Either way, merging this PR as it stands right now, I /think/ we'll have a bunch of pages (e.g. example blog posts) getting indexed where they previously weren't.

Reply:
Ah yes, I missed those, sorry! Both the no_index and noindex are being used apparently. Will let you know when it's fixed!